### PR TITLE
Create To-Do list structure

### PR DESCRIPTION
**Pull request by darrenodi**
In this branch I created a structure for my to-do-list using webpack

### CODE REVIEW

**Review by wandji20 — COMMENTED** [review]
## Hi @darrenodi  :raised_hands:

# Status: Invalid :stop_sign:

>Friendly reminder: You should submit a working project .<br> Make sure to write more descriptive commit messages that reflect the change you introduced <br> Make sure to write the steps to run your project locally(Just how to clone is not enough).  <br>If you cannot fix it by yourself, reach out to a full-time TSE. This [article](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-) describes how to set up a meeting with full-time TSEs.
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!


**Good luck :four_leaf_clover:** 

 _If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by darrenodi** [comment]
Greetings. But My project is working isn't it?

**Comment by darrenodi** [comment]
It's friday. Because of some other issues I had to wait till today to
submit Tuesday's task. I made sure everything is working but you say it is
not working?

**Comment by wandji20** [comment]
Hi @darrenodi,
I was not able to run a build command on your project, and your commit messages are not descriptive. Could you please fix these and and submit for another review?

**Comment by darrenodi** [comment]
That's because I am not using build. Use "npm run dev" .using build is not
a requirement. just like I'm not using run server.

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// import _ from 'lodash';
+import './style.css';
+// import { changeBox } from './component.js';
```
Comment (Dipeshtwis) [inline src/app.js:3]: - [ ] I would recommend removing all the commented code to make the codebase cleaner.

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+arrayTask.forEach(component);
+
+// const checkStatus = document.querySelectorAll('.checkbox');
+// checkStatus.forEach((item, index) => {
+//   item.addEventListener('change', () => {
+//     changeBox(item, arrayTask, index);
+//   });
+//   // item.addEventListener('click', changeBox(arrayTask,index));
+// });
```
Comment (Dipeshtwis) [inline src/app.js:44]: - [ ] Same here. I would recommend removing all the commented code to make the codebase cleaner.

File: src/component.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// function changeBox(item, givenArray, index) {
+//   if (item.checked === true) {
+//     givenArray[index].complete = true;
+//   } else {
+//     givenArray[index].complete = false;
+//   }
+//   console.log(givenArray);
+// }
+
+// export default changeBox;
```
Comment (Dipeshtwis) [inline src/component.js:10]: - [ ] Same here as well. I would recommend removing all the commented code to make the codebase cleaner.

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
+body {
+  /* background-color: bisque; */
```
Comment (Dipeshtwis) [inline src/style.css:2]: Same here as well. I would recommend removing all the commented code to make the codebase cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @darrenodi, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Webpack is used to set up the project.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @darrenodi, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**